src/mgmt/srv.c

```diff
@@ -300,6 +303,106 @@ process_join_request(Drpc__Call *drpc_req, Mgmt__JoinResp *resp)
 		resp->status = MGMT__DAOS_REQUEST_STATUS__ERR_UNKNOWN;
 }
 
+static void
+process_create_pool_request(Drpc__Call *drpc_req, Mgmt__CreatePoolResp *resp)
+{
+	Mgmt__CreatePoolReq	*pb_req = NULL;
+	d_rank_list_t		*targets = NULL;
+	d_rank_t		ranks[max_svc_nreplicas];
+	d_rank_list_t		svc = {};
+	d_rank_list_t		*svc_p = &svc;
```

Comment:
ds_mgmt_create_pool allocates the d_rank_list_t (plus its ranks array) and outputs its address. We don't need svc here. Something in the spirit of

    d_rank_list_t *svc;
    ...
    rc = ds_mgmt_create_pool(..., &svc);
    ...
    d_rank_list_free(svc);

will do.

Reply:
done